src/esssans/normalization.py

```diff
@@ -34,7 +34,7 @@
 
 
 def solid_angle(
-    data: CalibratedMaskedData[RunType],
+    data: SlicedMaskedData[RunType],
```

Comment:
I am a bit unhappy about this change. I think the previous version, which said "we compute the solid angle based on calibrated data" was clearly expressing the behavior and the intent of the pipeline. As it is now, it could be anything. If we later change the order of slicing and calibration this will be completely broken.

Reply:
Would simply changing the name to `SlicedCalibratedMaskedData` help or is that not what you mean?

Reply:
Would help partially. I still wonder though if this is correctly expressing the intent. Solid angle could be computed before slicing or after, so this choice is due to cost?